src/main/java/org/embulk/output/elasticsearch/ElasticsearchHttpClient.java

```diff
@@ -289,6 +312,34 @@ private void deleteIndex(String indexName, PluginTask task)
         }
     }
 
+    private void deleteAlias(String indexName, String aliasName, PluginTask task)
```

Comment:
I don't like to put the code used only for testing purposes here, please move it to the testing package.
Just my style, please ignore it if you think it is ok.

Reply:
It's used in `reassignAlias` method. So, it's not just for test only.